product.json

```diff
@@ -0,0 +1,189 @@
+/**
+ * Product domain object.
```

Comment:
What's the need for this file if you have product.js?

Reply:
Fixed